src/vm/method.hpp

```diff
@@ -3360,7 +3363,7 @@ class InstantiatedMethodDesc : public MethodDesc
         //
         // For generic method definitions that are not the typical method definition (e.g. C<int>.m<U>)
         // this field is null; to obtain the instantiation use LoadMethodInstantiation
-    PTR_Dictionary m_pPerInstInfo;  //SHARED
+    RelativePointer<PTR_Dictionary> m_pPerInstInfo;  //SHARED
```

Comment:
We'll split this pull request into two, leaving in this one only fields, which are not accessed from jit code